hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSEditLog.java

```diff
@@ -771,7 +771,7 @@ private void printStatistics(boolean force) {
         .append(" Total time for transactions(ms): ")
         .append(totalTimeTransactions)
         .append(" Number of transactions batched in Syncs: ")
-        .append(numTransactionsBatchedInSync.get())
+        .append(numTransactionsBatchedInSync.longValue())
```

Comment:
How frequently this gets printed compared to `longSync()`?

Reply:
At present, it is printed once a minute. I have not changed this frequency.

Reply:
> How frequently this gets printed compared to `longSync()`?

At present, it is printed once a minute. I have not changed this frequency.

Reply:
I see, Thanks @tomscut